index.html

```diff
@@ -304,13 +303,13 @@
       <h2><code>sendBeacon</code> Method</h2>
       <pre class="idl">
 partial interface <dfn id="Navigator">Navigator</dfn> {
-    boolean <a>sendBeacon</a>(<a>DOMString</a> <var title=
-"url">url</var>, optional (<a>ArrayBufferView</a> or <a>Blob</a> or <a>DOMString</a> or <a>FormData</a>)? <var>data</var> = null);
+    boolean <a>sendBeacon</a>(<a>USVString</a> <var title=
+"url">url</var>, optional <a>BodyInit</a>? <var>data</var> = null);
```

Comment:
I guess there's no way to get rid of `= null` at this point? Omitting the argument (or using undefined, which is identical) would be much cleaner.